.golangci.yml

```diff
@@ -6,6 +6,13 @@ linters-settings:
     statements: 60
   forbidigo:
     forbid:
+    - ioutil.Discard # use io.Discard
+    - ioutil.NopCloser # use io.NopCloser
+    - ioutil.ReadAll # use io.ReadAll
+    - ioutil.ReadFile # use os.ReadFile
+    - ioutil.TempDir # use os.MkdirTemp
+    - ioutil.TempFile # use os.CreateTemp
+    - ioutil.WriteFile # use os.WriteFile
```

Comment:
@jkowalski After we have moved `ioutil.ReadDir` to `os.ReadDir`, we can replace this with [depguard](https://golangci-lint.run/usage/linters/#depguard) to blacklist the `io/ioutil` package completely